flink-formats/flink-protobuf/src/main/java/org/apache/flink/formats/protobuf/util/PbFormatUtils.java

```diff
@@ -34,6 +35,15 @@ public static String getFullJavaName(Descriptors.Descriptor descriptor, String o
             String parentJavaFullName =
                     getFullJavaName(descriptor.getContainingType(), outerProtoName);
             return parentJavaFullName + "." + descriptor.getName();
+        } else if (descriptor.getFullName().contains("google")) {
```

Comment:
@hdulay The compilation error is caused by a third-party import, right? Could you make it more general to adapt to all the import cases?

Reply:
Yes